libraries/AP_HAL_ChibiOS/hwdef/f103-periph/hwdef.dat

```diff
@@ -57,8 +57,6 @@ PA5 SPI1_SCK SPI1
 PA6 SPI1_MISO SPI1
 PA7 SPI1_MOSI SPI1
 
-SPIDEV rm3100 SPI1 DEVID1 MAG_CS MODE0 1*MHZ 1*MHZ
```

Comment:
Why removing the RM3100 support ? There is no explanation why it was done in the commit message.

Reply:
I removed it from the general f103-periph and have now moved it into f103-GPS. We shouldn't be defining this in the generic fw which may not have a mag at all